src/pages/Sell/ListingPage.jsx

```diff
@@ -20,16 +61,45 @@ export default function ListingPage() {
       });
 
       setListing(data);
-      // TODO upload to database
-      navigate('/sell/confirmation/');
-    }
+
+      try {
+        // const auth = getAuth();
+        addDoc(collection(db, 'items'), {
```

Comment:
note the collection will be called `textbooks`

Reply:
Got it. I've noted this into the Progress Doc.